NotadogApi/Models/UserSignupCredentials.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace NotadogApi.Models
+{
+    public class UserSignupCredentials
```

Comment:
Some properties duplicates in **UserLoginCredentials**. Consider using inheritance.